### PR TITLE
Allow more customization of forest and pairplot

### PR DESCRIPTION
**Pull request by ColCarroll**
This makes the forestplot layout a little nicer, and allows adjusting marker size in `plot_pair`.

### CODE REVIEW

File: arviz/plots/forestplot.py [hunk 1]
```diff
@@ -107,7 +107,9 @@ def plot_forest(data, kind='forestplot', model_names=None, var_names=None, combi
                              ncols=ncols,
                              figsize=figsize,
                              gridspec_kw={'width_ratios': width_ratios},
-                             sharey=True)
+                             sharey=True,
+                             constrained_layout=True,
+                             )
```
Comment (canyon289) [inline arviz/plots/forestplot.py]: You and @aloctavodia have very different styles on end parentheses and braces :)

**Comment by canyon289** [comment]
LGTM

File: arviz/plots/forestplot.py (hunk 1 above)
Comment (ColCarroll) [reply arviz/plots/forestplot.py]: oy, I don't care one way or another, but know very nice people who do, and sometimes forget which convention I should be working with in which project!  if we can find the right pylint/black/pycodestyle check for that, happy to merge it.

Comment (aloctavodia) [reply arviz/plots/forestplot.py]: The dance of the lonely parenthesis.

**Pull request merged**